escape.go

```diff
@@ -210,140 +201,108 @@ func (ei *escapeInterpreter) parseOne(ch rune) (isEscape bool, err error) {
 	return false, nil
 }
 
-// outputNormal provides 8 different colors:
-//   black, red, green, yellow, blue, magenta, cyan, white
-func (ei *escapeInterpreter) outputNormal() error {
-	for _, param := range ei.csiParam {
-		p, err := strconv.Atoi(param)
+func (ei *escapeInterpreter) outputCSI() error {
```

Comment:
Could we add some comments here explaining the significance of each case? For example it's not clear to me what the significance of `p == 38` is.

Reply:
Actually I just realised we have some constants defined at the top of the file that we can use here e.g. `setForegroundColor = 38`

Reply:
added comments and changed to use constants.